Sources/GDSCommon/Extensions/UIKit/UIViewController+Extensions.swift

```diff
@@ -2,7 +2,9 @@ import UIKit
 
 extension UIViewController {
     func setBackButtonTitle(isHidden: Bool = false) {
-        navigationItem.backButtonTitle = NSLocalizedString(key: "backButtonTitle")
+        let backButtonTitle = GDSLocalisedString(stringKey: "GDSCommon-backButtonTitle",
```

Comment:
Shouldn't our localised string keys be camelCase rather than snake case?

Or perhaps we should agree a _common_ prefix such as Apple uses for UIKit classes  (eg, `UI`) for our gds common types and properties?